delete_page.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+
+"""
+    delete_page.py
+
+    MediaWiki Action API Code Samples
+    'Delete' module:
+    1st Step;Fetching token of type `csrf`
+    2nd Step;Used fetched 'csrf_token' to delete a page
+    MIT license
+"""
+
+import requests
+
+S = requests.Session()
+
+URL = "https://test.wikipedia.org/w/api.php"
+
+PARAMS_0 = {
+    'action':"query",
+    'meta':"tokens",
+    'type':"csrf",
+    'format':"json"
+}
+
+R = S.get(url=URL, params=PARAMS_0)
+DATA = R.json()
+
+CSRF_TOKEN = DATA['query']['tokens']['csrftoken']
+
+print(CSRF_TOKEN)
+
+#Use the csrf token in a post request to delete a page
+
+PARAMS_1 = {"action":"delete", "title":"Rabosot", "token":CSRF_TOKEN, "format":"json"}
```

Comment:
I had to arrange PARAMS_1 that way because after running it in Pylint, it complained of "trialing whitespace", at the level of the parameters in PARAMS_1.